package.json

```diff
@@ -8,7 +8,11 @@
 		"test": "echo \"Error: no test specified\" && exit 1",
 		"lint": "yarn workspace @microsoft/atlas-css lint",
 		"lint:fix": "yarn lint --fix",
+		"start": "npm-run-all --parallel serve:*",
```

Comment:
Just me or is `ctrl+c` not working well?

Reply:
aside- you likely will not need to run things in parallel because the site package will take a dependency on the css package and build the css as part of it's process to build it's own css

Reply:
npm-run-all removed - it was working spottily for me too